README.md

```diff
@@ -15,29 +15,29 @@ as **T-F-D**:
 ## Types
 
  | Type code   | Referencing        |
- | ----------- | ------------------ |
- | 0           | Peer ID            |
- | 1           | Message ID         |
- | 2           | Blob ID            |
- | 3           | Diffie-Hellman key |
- | 4           | Signature          |
- | 5           | Encrypted data     |
- | 6           | Generic data       |
-
-### Peer ID formats
-
-A peer ID TFD represents the public portion of a cryptographic keypair used to
+ |:-----------:| ------------------ |
+ | 0           | feed               |
+ | 1           | msg                |
+ | 2           | blob               |
+ | 3           | key                |
+ | 4           | signature          |
+ | 5           | encrypted          |
+ | 6           | generic            |
+
+### 0. feed formats
+
+A feed ID TFD represents the public portion of a cryptographic keypair used to
 identify a peer, and in turn identify feeds. Note however that there are some
 identities that do not have a feed nor create messages, such as Fusion
 Identities.
 
 | Type code | Format code | Data length | Format name     | Specification    |
-|-----------|-------------|-------------|-----------------|------------------|
-| 0         | 0           | 32 bytes    | Classic         | [classic]        |
-| 0         | 1           | 32 bytes    | Gabby Grove     | [gabby grove]    |
-| 0         | 2           | 32 bytes    | Bamboo          | [bamboo]         |
-| 0         | 3           | 32 bytes    | Bendy Butt      | [bendy butt]     |
-| 0         | 4           | 32 bytes    | Fusion Identity | [fusionidentity] |
+|:---------:|:-----------:|-------------|-----------------|------------------|
+| 0         | 0           | 32 bytes    | classic         | [classic]        |
+| 0         | 1           | 32 bytes    | gabby-grove     | [gabby grove]    |
```

Comment:
I assume you introduced the hyphen so to make this match bfe.json better? Still, I think this specification is a formal spec, not code, so it should use uppercase. When making a PR, you could also give me some context on why this needed to be changed to lowercase.